integration_tests/pages/goal/AddStepPage.ts

```diff
@@ -46,9 +55,11 @@ export default class AddStepPage extends Page {
 
   targetDateYearField = (): PageElement => cy.get('#targetDate-year')
 
-  addStepButton = (): PageElement => cy.get('#add-step-button')
+  addAnotherStepButton = (): PageElement => cy.get('#add-another-step-button')
 
   submitButton = (): PageElement => cy.get('#submit-button')
 
   prisonNumberLabel = (): PageElement => cy.get('[data-qa=prison-number]')
+
+  stepNumberLabel = (): PageElement => cy.get('label[for=title]')
```

Comment:
You can explain this one to me tomorrow 🤯

Reply:
lol - it's a css selector (that's what all the args in the `cy.get` method calls are - they are all css selectors to select 1 or more elements on the page)
Specifically its the selector to get a `label` element whose attribute `for` has the value `title`. IE. it selects this element in the markup:
```
<label class="govuk-label govuk-label--m" for="title">
```
![Screenshot 2023-07-06 at 08 41 13](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/assets/94835226/938da721-7dc2-4b18-a26d-14221226e638)